pkg/controller/master/backup/restore.go

```diff
@@ -755,13 +758,16 @@ func (h *RestoreHandler) deleteOldPVC(vmRestore *harvesterv1.VirtualMachineResto
 }
 
 func (h *RestoreHandler) startVM(vm *kubevirtv1.VirtualMachine) error {
-	runStrategy, _ := vm.RunStrategy()
-	logrus.Infof("starting the vm %s, current state running:%v", vm.Name, runStrategy)
+	runStrategy, err := vm.RunStrategy()
+	if err != nil {
+		return err
+	}
 
+	logrus.Infof("starting the vm %s, current state running:%v", vm.Name, runStrategy)
 	switch runStrategy {
-	case kubevirtv1.RunStrategyAlways:
+	case kubevirtv1.RunStrategyAlways, kubevirtv1.RunStrategyRerunOnFailure:
 		return nil
-	case kubevirtv1.RunStrategyManual, kubevirtv1.RunStrategyRerunOnFailure:
+	case kubevirtv1.RunStrategyManual:
```

Comment:
This will cause,
If we execute poweroff command inside VM, it will never have "Start button" but only have "restart button".
We could only "restart" it.
Do we expect this behavior?

If yes, this patch is good to me, just ignore this comment.

Reply:
Actually I found users is still be able to have the `Start` button even if we shutdown the VM inside the guestOS, check the [canStart action formatter](https://github.com/harvester/harvester/blob/master/pkg/api/vm/formatter.go#L147)
<img width="2321" alt="image" src="https://user-images.githubusercontent.com/4569037/165921258-19c39567-9b7f-4459-967f-3d0ac603746c.png">

Reply:
But if VM is RerunOnFailure, it will never have start button.
and RerunOnFailure VM is able to shutdown from inside.

Reply:
hmm, wired we can check offline if u have env that can reproduce this behavior, at least it works in my env, thanks.
<img width="2258" alt="image" src="https://user-images.githubusercontent.com/4569037/165922340-2cb803bd-867a-4e92-b372-c7bd03ced615.png">

Reply:
yep, if we have VMI, we should let users backup it.